### PR TITLE
Feature packaging

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -38,11 +46,11 @@
...
     zip_safe=False,
-    keywords='gitenberg.metadata',
+    keywords="books ebooks gitenberg gutenberg epub metadata",
     classifiers=[
-        'Development Status :: 2 - Pre-Alpha',
+        'Development Status :: 3 - Alpha',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
+        'License :: OSI Approved :: GNU General Public License (GPL)',
         'Natural Language :: English',
```
Comment (eshellman) [inline setup.py:55]: LGPL might be a better choice here

File: setup.py [hunk 2]
```diff
@@ -15,22 +15,30 @@
...
 
 
 setup(
     name='gitenberg.metadata',
-    version='0.1.0',
+    version='0.1.4',
     description="metadata development - formats and machinery for GITenberg",
     long_description=readme + '\n\n' + history,
     author="Raymond Yee",
     author_email='raymond.yee@gmail.com',
```
Comment (eshellman) [inline setup.py]: maybe add me as author?

Comment (sethwoodworth) [inline setup.py]: This was autofilled when Raymond generated this setup.py file.  I'll add you as the author, idk how to add multiple authors other than an AUTHORS file

File: setup.py [hunk 3]
```diff
@@ -15,22 +15,30 @@
...
     name='gitenberg.metadata',
-    version='0.1.0',
+    version='0.1.5',
     description="metadata development - formats and machinery for GITenberg",
     long_description=readme + '\n\n' + history,
-    author="Raymond Yee",
-    author_email='raymond.yee@gmail.com',
-    url='https://github.com/rdhyee/gitenberg.metadata',
+    author="Eric Hellman",
+    author_email='eric@hellman.net',
```
Comment (sethwoodworth) [inline setup.py:40]: Updated with author info.  When we figure out how to add multiple authors we can throw ourselves into an AUTHORS.rst file

File: setup.py [hunk 4]
```diff
@@ -38,11 +46,12 @@
...
     zip_safe=False,
-    keywords='gitenberg.metadata',
+    keywords="books ebooks gitenberg gutenberg epub metadata",
     classifiers=[
-        'Development Status :: 2 - Pre-Alpha',
+        'Development Status :: 3 - Alpha',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
+        'Intended Audience :: Education',
+        'License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)',
```
Comment (sethwoodworth) [inline setup.py:54]: Changed license

**Pull request merged**